apiserver/apiserver.go

```diff
@@ -937,7 +972,7 @@ func (srv *Server) serveConn(
 		for apiVersion, factory := range adminAPIFactories {
 			adminAPIs[apiVersion] = factory(srv, h, apiObserver)
 		}
-		conn.ServeRoot(newAdminRoot(h, adminAPIs), serverError)
+		conn.ServeRoot(newAdminRoot(h, adminAPIs), nil, serverError)
```

Comment:
we don't need the recorder here because we aren't doing anything yet, just setting up?

Reply:
No, that code is wrong now - it needs to be a recorder factory with at least the apiObserver there.